test/utils/recordUtils.js

```diff
@@ -31,9 +29,8 @@ export const getNodePath = node => (survey, record) => {
 
     if (NodeDef.isMultiple(nodeDef)) {
       const siblings = Record.getNodeChildrenByDefUuid(parentNode, nodeDefUuid)(record)
-      const index = R.findIndex(n => Node.getUuid(n) === Node.getUuid(node), siblings)
-      const position = index + 1
-      return `${parentNodePath}/${NodeDef.getName(nodeDef)}[${position}]`
+      const index = R.findIndex((n) => Node.getUuid(n) === Node.getUuid(node), siblings)
+      return `${parentNodePath}/${NodeDef.getName(nodeDef)}[${index}]`
```

Comment:
before the text has:
[${position}] and position was index + 1
now the text id:
[${index}]

is still the same?

Reply:
no...! I've changed it to be consistent with the way we access nodes now (zero based). Anyway, this module is used only for tests... probably other integration tests will be broken by these change, but they need in any case to be refactored.